templates/terraform/nested_property_documentation.erb

```diff
@@ -4,9 +4,9 @@
 -%>
 The `<%= Google::StringUtils.underscore(property.name) -%>` block <%= if property.output then "contains" else "supports" end -%>:
 <% nested_properties.each do |prop| -%>
-<%= build_property_documentation(prop) -%>
+<%= build_property_documentation(prop) %>
```

Comment:
https://github.com/GoogleCloudPlatform/magic-modules/blob/master/GOVERNANCE.md#prefer---does-not-add-n-to-